src/lib/network_pool/transaction_pool.ml

```diff
@@ -2164,6 +2170,9 @@ let%test_module _ =
         }
       in
       let parties = Transaction_snark.For_tests.multiple_transfers test_spec in
+      let (`If_this_is_used_it_should_have_a_comment_justifying_it parties) =
```

Comment:
This should have a comment justifying it.

Reply:
chnaged it to `to_valid` but it'll will fail if the use is unintended. Also changed the name to make it clearer